nucliadb/nucliadb/search/api/v1/resource.py

```diff
@@ -101,6 +99,12 @@ async def search(
     debug: bool = Query(False),
     shards: List[str] = Query(default=[]),
 ) -> ResourceSearchResults:
+    if not all(map(lambda feature: feature == SearchOptions.PARAGRAPH, features)):
+        raise HTTPException(
+            status_code=422,
+            detail=f"Invalid search option. Only {SearchOptions.PARAGRAPH} is valid",
+        )
```

Comment:
If there is only one option, why providing the parameter at all?
I am in for removing the param and returning paragraphs always. Then we don't need this extra logic.
If we happen to offer more features in the future we can add it back easily.
WDYT?

Reply:
Seems fine to me!